.github/actions/e2e_benchmark/evaluate/compare.py

```diff
@@ -93,39 +99,54 @@ def main() -> None:
         '- Commit of current benchmark: {ch}'.format(ch=bench_curr['commit']),
         '- Commit of previous benchmark: {ch}'.format(ch=bench_prev['commit']),
         '',
-        '| Benchmark suite | Current | Previous | Ratio |',
-        '|-|-|-|-|',
+        '| Benchmark suite | Metric | Current | Previous | Ratio |',
```

Comment:
Can we link the commit in the table header, like [Previous](https://github.com/edgelesssys/constellation/commit/96b4b74a7a77fc2003a395ade1f9969232ff80d1)?